packages/widgets/src/types.ts

```diff
@@ -32,7 +32,12 @@ export type WidgetSettings = {
   /** Widget subtitle */
   subtitle: TextSettings;
   /** Legend component settings */
-  legend?: LegendSettings;
+  legend?: LegendSettings & BubbleWidgetLegendSettings;
```

Comment:
As in me previous comment it should be alternative - not union - `LegendSettings | BubbleWidgetLegendSettings `

Reply:
Sorry for that, now should be corrected